README.md

```diff
@@ -2,10 +2,43 @@
 
 ## Ingress Istio Controller
 
-Configure Istio Virtual Services objects from Ingress objects.
-
 Based on https://github.com/kubernetes/sample-controller
 
+This controller is a reimplementation of the logic implemented by the [Istio Ingress](https://istio.io/latest/docs/tasks/traffic-management/ingress/kubernetes-ingress/)
+which creates VirtualServices based on Ingress objects and routes their traffic through a Gateway. 
+
+This reimplementation is du to that fact that in Istio 1.6, the core logic changed and instead of all of the generated VirtualServices routing through a single 
```

Comment:
```suggestion
This reimplementation is due to that fact that in Istio 1.6, the core logic changed and instead of all of the generated VirtualServices routing through a single 
```